pyproject.toml

```diff
@@ -70,11 +72,11 @@ np = "nanopub.__main__:cli"
 
 
 [project.urls]
-Homepage = "https://nanopublication.github.io/nanopub-py/"
-Documentation = "https://nanopublication.github.io/nanopub-py/"
-History = "https://github.com/Nanopublication/nanopub-py/releases"
-Tracker = "https://github.com/Nanopublication/nanopub-py/issues"
-Source = "https://github.com/Nanopublication/nanopub-py"
+Homepage = "https://fair-workflows.github.io/nanopub"
+Documentation = "https://fair-workflows.github.io/nanopub"
+History = "https://github.com/fair-workflows/nanopub/releases"
+Tracker = "https://github.com/fair-workflows/nanopub/issues"
+Source = "https://github.com/fair-workflows/nanopub"
```

Comment:
These links redirect to `https://github.com/Nanopublication/nanopub-py/…` as far as I can see; I am not sure what `fair-workflows` is. Maybe more context would be needed?

Reply:
I made a comment, that this is to be considered a draft PR. I need it to be labelled a conventional PR, for otherwise the pipelines will not run. The toml file is huge and contains numerous issues. I will work on it as time permits.